libraries/mediaupload/impl/src/main/kotlin/io/element/android/libraries/mediaupload/ImageCompressor.kt

```diff
@@ -37,17 +37,18 @@ class ImageCompressor @Inject constructor(
 
     /**
      * Decodes the [inputStream] into a [Bitmap] and applies the needed transformations (rotation, scale) based on [resizeMode], then writes it into a
-     * temporary file using the passed [format] and [desiredQuality].
+     * temporary file using the passed [format], [orientation] and [desiredQuality].
      * @return a [Result] containing the resulting [ImageCompressionResult] with the temporary [File] and some metadata.
      */
     suspend fun compressToTmpFile(
         inputStream: InputStream,
         resizeMode: ResizeMode,
         format: Bitmap.CompressFormat = Bitmap.CompressFormat.JPEG,
+        orientation: Int = 0,
```

Comment:
Maybe use named const instead of 0.
```suggestion
        orientation: Int = ExifInterface.ORIENTATION_UNDEFINED,
```
or just remove the default parameter value.